libretro-common/include/libretro.h

```diff
@@ -976,6 +976,19 @@ struct retro_hw_render_context_negotiation_interface
                                             * so it will be used after SET_HW_RENDER, but before the context_reset callback.
                                             */
 
+struct retro_resource
+{
+   const char *rel_path;
+   const char *base_path;
```

Comment:
in my libretro wrapper, I cache all the paths so that a static value can be returned. I much prefer a char* + size_t, but where should the free take place?